xAPI.md

```diff
@@ -2639,34 +2658,48 @@ such a mechanism.
 * The LRS SHOULD at a minimum supply OAuth with "HMAC-SHA1" and "RSA-SHA1" signatures.
 
 ###### Application registered + known user Process and Requirements
+**Process:** The standard workflow for OAUth 1.0 is used. 
 
-* Use endpoints in section [6.4.2 OAuth Authorization Scope](#oauthscope) to complete the standard OAuth workflow 
+**Requirements:**
+* The LRS MUST support the endpoints in section [6.4.2 OAuth Authorization Scope](#oauthscope) to complete the standard OAuth workflow 
 (details not in this specification).
```

Comment:
Should we instead say where these ARE detailed? http://tools.ietf.org/html/rfc5849